source/online_usecases/small_circuit_in_silico_experiments/pair_recording_app/pair_recording_app.rst

```diff
@@ -0,0 +1,97 @@
+==========================
+Pair Recording application
+==========================
+
+Pair Recording application is a web environment for a small circuit
+in-silico experiments.
+
+It provides an extended toolset to configure and run simulations of single cell
+models as well as multiple connected neurons from a given circuit.
+The application's graphical user interface allows user to define in a first
+step a small circuit composed of several connected neurons, and in a second
+step to experiment on this circuit by defining a stimulation and reporting
+protocol.
+
+Usage
+=====
+
+The use case can be found in `Online Use Cases` under
+`Small Circuit In Silico Experiments` menu in the `Brain Simulation Platform`.
+A user can select a base circuit from the list to open Pair Recording
+application.
+
+.. image:: images/step_1.png
+   :width: 800 px
+
+Usage flow usually consists of:
+
+#. selecting a small circuit of several connected neurons
+#. configuring stimulation and reporting protocols
+#. running simulation and inspecting its results
+
+When the application has been started (on the initial load it will fetch
+necessary circuit information, e.g. circuit info, cell locations and
+properties, so this might take some time) an interactive circuit visualisation
+will appear with controls facilitating cell selection.
+
+.. image:: images/step_2.png
+   :width: 800 px
+
+Cells in the view are colour-coded by one of their's properties, user can
+change this property with a `Color by` select on the bottom of the view.
+Form to the right consists of various controls and allows user to filter and
+select cells. Two types of filters are available: property and connectivity.
+
+Property(display) filters can be used to show only cells with particular
+properties (include subtype) or filter out a subset of particular cells based
+on their properties. By default, multiple added filters use OR (union) logic to
+compose a subset of affected cells, and when required,  user can switch this
+mode to use AND (intersection) logic. This applies to both types of filters:
+`include` and `exclude`. Connectivity filter can be used to show neurons with
+afferent of efferent connections to a selected cell.
+
+Cells can be selected for later simulations by clicking on them in the
+interactive view or, if known beforehand, specifying their `gid`s in the form
+and click the `Add` button.
+
+.. image:: images/step_3.png
+   :width: 800 px
+
+When cell selection is done a user can proceed to simulation configuration by
+clicking a corresponding button which will load and render morphologies for
```

Comment:
... morphologies for **the** selected cells.

Reply:
Done